include/openthread/srp_server.h

```diff
@@ -167,6 +167,32 @@ typedef struct otSrpServerLeaseConfig
     uint32_t mMaxKeyLease; ///< The maximum KEY-LEASE interval in seconds.
 } otSrpServerLeaseConfig;
 
+/**
+ * This structure includes SRP server lease information of a host/service.
+ *
+ */
+typedef struct otSrpServerLeaseInfo
+{
+    uint32_t mLease;             ///< The lease time of a host/service in seconds.
+    uint32_t mKeyLease;          ///< The key lease time of a host/service in seconds.
+    uint32_t mRemainingLease;    ///< The remaining lease time of the host/service in milliseconds.
+    uint32_t mRemainingKeyLease; ///< The remaining key lease time of a host/service in milliseconds.
```

Comment:
Can we have the same time unit? All use `seconds` or `milliseconds`?

Reply:
Updated to use all milliseconds